packages/portal-proto/src/components/Table/VerticalTable.tsx

```diff
@@ -273,73 +273,77 @@ function VerticalTable<TData>({
 
   return (
     <div data-testid={customDataTestID} className="grow overflow-hidden">
-      <div
-        className={`flex flex-wrap gap-y-4 mb-2 ${
-          !additionalControls ? "justify-end" : "justify-between"
-        }`}
-      >
-        {additionalControls && <>{additionalControls}</>}
-        <div className="flex flex-wrap gap-y-2 gap-x-4 items-center">
-          {tableTitle && (
-            <Text className="self-center uppercase text-lg text-left ml-0 lg:ml-auto">
-              {tableTitle}
-            </Text>
-          )}
+      {(additionalControls ||
+        tableTitle ||
+        search?.enabled ||
+        showControls) && (
+        <div
```

Comment:
Just a style thing, I think this would be more readable if the additional controls, table title etc. were broken out into a separate component.  It's kind of hard to figure out what this conditional is rendering when it spans like ~75 lines.

Reply:
I updated the code.